Doc/library/math.rst

```diff
@@ -136,6 +136,15 @@ Number-theoretic and representation functions
    .. versionadded:: 3.5
 
 
+.. function:: lcm(a, b)
+
+   Return the least common multiple of integers *a* and *b*.  If either *a* or
```

Comment:
This statement isn't right: `lcm(a, b)` is only positive in the case where *both* of `a` and `b` are nonzero.

Suggest replacing "either" with "both", and adding a new sentence explaining what happens if either of the inputs is zero.